src/environments/environment.prod.ts

```diff
@@ -16,3 +16,14 @@ export const environment = {
   DB_PROXY_PREFIX: "/db",
   DB_NAME: "app",
 };
+
+export const firebaseConfig = {
+  apiKey: "AIzaSyAS-AmoAzv_-bRbvm1MMqKaq8t0xjZ7Wqo",
+  authDomain: "aam-digital-b8a7b.firebaseapp.com",
+  projectId: "aam-digital-b8a7b",
+  storageBucket: "aam-digital-b8a7b.firebasestorage.app",
+  messagingSenderId: "189059495005",
+  appId: "1:189059495005:web:c6bdb0c8c665864b37c9b4",
+  vapidKey:
+    "BKkE6EgJBIwRa9-DUSKZpmkMuG7Fak2lZgxda_DPx5kYkeK8cgQM_xqurHqxRNa1b2MuW7-_t9iFbgfuXUsWF5I",
+};
```

Comment:
This probably should not be part of the public repository (I hope these do not include live system secrets?).
We have struggled a bit to find a really good way to store and deploy such config but maybe we can put this into a json file in the assets folder (and on the gitignore list?) and load it from there?

Reply:
Makes sense. For this, first, I created a file named firebase-config.json inside the src/assets folder, but when I tried to load that file, it wasn’t loading properly, which caused the Firebase app initialization to fail. Then I created a new file firebase-config.ts, exported that file, and used it directly. This worked fine, and I also added this file to the .gitignore. Please take a look and let me know if any changes are needed.

Reply:
I don't think this gives us the full flexibility we need: There must be an option to set these config values without re-building the angular app.

What errors did you receive when trying to load a json file from the assets folder? 
Is the way we do this for our core config.json file here maybe an option to copy for another json file? https://github.com/Aam-Digital/ndb-core/blob/f4ffa7ceb4a631199087755e54e0c137f278ba3c/src/bootstrap-environment.ts#L4

Reply:
Thanks for the suggestion. Now, I've initialized the `firebase-config.json` inside the `bootstrap-environment.ts` file, and then added the `firebaseConfig` to the environment. So, when the `bootstrap-environment.ts` is loaded, it adds all the data from the `firebase-config.json` file into it, and I am directly accessing it from the environment. This way, it's working fine.

Reply:
Please take a look and let me know if there are any changes needed.